src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -1164,17 +1162,11 @@ export default class Blockchain extends Emittery<BlockchainTypedEvents> {
     }
   }
 
-  #traceTransaction = async (
-    transaction: VmTransaction,
+  #fastForwardVM = async (
+    transactionIndex: number,
     trie: GanacheTrie,
-    newBlock: RuntimeBlock & { transactions: VmTransaction[] },
-    options: TraceTransactionOptions,
-    keys?: Buffer[],
-    contractAddress?: Buffer
-  ): Promise<TraceTransactionResult> => {
-    let currentDepth = -1;
-    const storageStack: TraceStorageMap[] = [];
-
+    block: RuntimeBlock & { transactions: VmTransaction[] }
+  ): Promise<VM & { stateManager: GanacheStateManager }> => {
```

Comment:
`traceTransaction` previously had `debug_storageRangeAt`-specific logic in it, which was just gross and weird. I've refactored our the common code into `fastForwardVM`.